packages/ng/libraries/core/src/lib/user/select/input/user-select-input.component.html

```diff
@@ -13,5 +13,9 @@
 </ng-template>
 <lu-option-picker-advanced>
 	<lu-user-paged-searcher></lu-user-paged-searcher>
-	<lu-option *luForOptions="let user" [value]="user">{{ user | luUserDisplay: searchFormat }}</lu-option>
+	<lu-user-homonyms></lu-user-homonyms>
+	<lu-option *luForOptions="let user" [value]="user">
+		<div>{{ user | luUserDisplay: searchFormat }}</div>

```

Comment:
Shouldn't `luUserDisplayFormat` be an input of `lu-user-homonyms` ? Otherwise, you could display users without detecting homonyms